Source/WebKit/WebProcess/GPU/webrtc/MediaRecorderProvider.cpp

```diff
@@ -41,8 +41,10 @@ std::unique_ptr<WebCore::MediaRecorderPrivate> MediaRecorderProvider::createMedi
 {
 #if ENABLE(GPU_PROCESS)
     auto* page = m_webPage.corePage();
-    if (page && page->settings().webRTCPlatformCodecsInGPUProcessEnabled())
-        return makeUnique<MediaRecorderPrivate>(stream, options);
+    if (page && page->settings().webRTCPlatformCodecsInGPUProcessEnabled()) {
+        // FIXME: It seems wrong to return a refcounted object in a unique_ptr.
+        return makeUniqueWithoutRefCountedCheck<MediaRecorderPrivate>(stream, options);
```

Comment:
I must have missed something, but I don't see neither WebKit::MediaRecorderPrivate nor WebCore::MediaRecorderPrivate having a ref/deref method.
It does inherit from `RealtimeMediaSource::AudioSampleObserver` and `RealtimeMediaSource::VideoFrameObserver` but they don't either. (Only `RealtimeMediaSource` itself has ref()/deref() method.